docker-cloud/main_test.go

```diff
@@ -0,0 +1,64 @@
+package main_test
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/ory/dockertest/v3"
+)
+
+var pool *dockertest.Pool
+var resource *dockertest.Resource
+
+func TestMain(m *testing.M) {
+	var err error
+	pool, err = dockertest.NewPool("")
+	if err != nil {
+		log.Fatalf("Could not connect to docker: %s", err)
```

Comment:
```suggestion
		log.Fatalf("Could not connect to docker daemon: %s", err)
```

To try to make clear it's not the dockerised application that's failing